README.md

```diff
@@ -282,7 +282,7 @@ How to choose between all these features? Here are some considerations:
     }
     ```
```

Comment:
Let's add a section (somewhere at the start of doc) about validation and how to disable it.

Some benchmarks significantly suffer from validation:
```
Lz4/false	decoded= 560000000	received= 289251866	thrpt=    989.89 MiB/s	time=539.511632ms
Lz4/true	decoded= 560000207	received= 289251903	thrpt=    743.22 MiB/s	time=718.572298ms
None/false	decoded= 560000000	received= 560000000	thrpt=   3900.61 MiB/s	time=136.916286ms
None/true	decoded= 560000207	received= 560000207	thrpt=   1004.50 MiB/s	time=531.663639ms
```

So, I would like to tell people how to disable it and write that the performance gain can be between x1.1 and x3, depending on the data.

Reply:
@slvrtrn I would like to push the bench for the provided results (it's L2 market data), is it ok?